content/en/contributing/terraform-tests.md

```diff
@@ -7,4 +7,4 @@ aliases:
   - /developer-guide/terraform-tests/
 ---
 
-We regularly run the test suite of the Terraform AWS provider against LocalStack to test the compatibility of LocalStack to Terraform. To achieve that, we have a dedicated [GitHub action](https://github.com/localstack/localstack/blob/master/.github/workflows/terraform-tests.yml) on [LocalStack](https://github.com/localstack/localstack), which executes the allow listed set of tests of [hashicorp/terraform-provider-aws](https://github.com/hashicorp/terraform-provider-aws/).
+We regularly run the test suite of the Terraform AWS provider against LocalStack to test the compatibility of LocalStack to Terraform. To achieve that, we have a dedicated [GitHub action](https://github.com/localstack/localstack-terraform-test/blob/main/.github/workflows/main.yml) on [LocalStack](https://github.com/localstack/localstack), which executes the allow listed set of tests of [hashicorp/terraform-provider-aws](https://github.com/hashicorp/terraform-provider-aws/).
```

Comment:
nit: It would be good to rephrase the sentence here a bit. The "on LocalStack" part here is a bit incoherent if the pipeline is in a different repo / the links point towards two different repos.